test/README.md

```diff
@@ -97,12 +97,11 @@ TEST_OUTPUT:
 Hello, World!
 ---
 */
-import std.stdio;
```

Comment:
Using Phobos in the testsuite is more or less an anti-pattern, so imho it's a good idea if the example doesn't use Phobos.